charts/castai-agent/templates/deployment.yaml

```diff
@@ -57,16 +57,15 @@ spec:
             - secretRef:
                 name: {{ include "castai-agent.fullname" . -}}
           {{/* we don't set resources if Cluster proportional vertical autoscaler is enabled */}}
-          {{- if not .Values.clusterVPA.enabled }}
           resources:
-            {{- .Values.resources | toYaml | nindent 14 }}
-          {{- else }}
+          {{- if .Values.clusterVPA.enabled }}
           {{/* we hardcode CPU req/limit for now */}}
-          resources:
             requests:
               cpu: 100m
             limits:
               cpu: 1000m
+          {{- else }}
+            {{- .Values.resources | toYaml | nindent 14 }}
```

Comment:
nitpick: people usually use `{{- else -}}` instead of adding `{{-` to the line below.